ansible/roles/test/files/ptftests/vxlan-decap.py

```diff
@@ -46,7 +46,8 @@ def count_matched_packets_helper(test, exp_packet, exp_packet_number, port, devi
         raise Exception("%s() requires positive timeout value." % sys._getframe().f_code.co_name)
 
     total_rcv_pkt_cnt = 0
-    while True:
+    end_time = time.time() + timeout
+    while time.time() < end_time:
```

Comment:
maybe we can look for other places that has similiar pattern

Reply:
At least this file is clean. I looked when I made the change. But checking other files would be good too.